### DISCUSSION TITLE
Vike StrictMode

### DISCUSSION THREAD

**Discussion by enri90** [vikejs/vike#1631] [Discussion]
### Description

Hi I'm having a problem with useEffect in strict-mode dev mode it reloads twice and I can't find the cause I tried to reproduce and the same problem can you analyse?

```typescript
pnpm create @batijs/app --react --tailwindcss --eslint --prettier
``` 

https://stackblitz.com/edit/u36ilv?file=package.json,pages%2Findex%2F%2BPage.tsx
https://react.dev/reference/react/StrictMode#enabling-strict-mode-for-entire-app

```typescript
  useEffect(() => {
    console.log('useEffect');
    fetch('https://api.imgflip.com/get_memes')
      .then((res) => res.json())
      .then((data) => setAllMemeImages(data.data.memes));
  }, []);
``` 

<img width="714" alt="Screenshot 2024-05-01 alle 22 33 31" src="https://github.com/vikejs/vike/assets/2630335/16346097-4d7b-4af7-89e3-c3e50367606d">

thx

**Comment by brillout** [comment]
This doesn't seem related to Vike.

Please respect https://brillout.github.io/rules/ (or use Discord) and don't mis-use GitHub issues for user-land questions. Thank you.

**Reply by enri90** [reply]
Discord doesn't work I don't get a response

I keep trying other solutions

Thanks

**Reply by brillout** [reply]
We've added a Help Guide and see https://vike.dev/faq#how-can-i-reach-out-for-help.
